JPS_BASE_LIB/python_wrapper/README.md

```diff
@@ -423,6 +423,51 @@ def doTask2():
 
  The py4jps aim is to provide Python access to the `TheWorldAvatar` project classes and methods. However, it is important to understand that not all `TheWorldAvatar` classes can be accessed. Namely, **any servlet depending classes can not be instantiated in Python without running the TomCat server first**. Since this has not been tested, it is not guaranteed that running the TomCat server would fix the problem. However, this should not be an issue for the `py4jps` users, given that the main purpose of the wrapper is to use the client-side `TheWorldAvatar` code to perform KG queries or updates. In other words, it is not the `py4jps` purpose to develop the server-side code, which should happen in Java.
 
+# Package release
+
+Maintainers who package and publish the most up-to-date codes from the `develop` branch handle the distribution of package py4jps on PyPI and Test-PyPI. If you want to release the package independently, i.e. become a maintainer, please contact the repository's administrator to indicate your interest.
+
+The release procedure is currently semi-automated and requires a few items:
+
+- Your Test-PyPI and PyPI account and password
+- The version number x.x.x for the release
+- The absolute path to `TheWorldAvatar` project folder on your machine
```

Comment:
I would simply say that it is required to have the `TheWorldAvatar` repository on your local machine.